Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -15146,6 +15146,24 @@ void WebPageProxy::requestTargetedElement(const API::TargetedElementRequest& req
     });
 }
 
+void WebPageProxy::requestAllTargetableElements(WebCore::FloatSize minimumSize, CompletionHandler<void(const Vector<Vector<Ref<API::TargetedElementInfo>>>&)>&& completion)
```

Comment:
Make the completion handler argument an rvalue ref too? Since you're creating a new `Vector` anyway.

```suggestion
void WebPageProxy::requestAllTargetableElements(WebCore::FloatSize minimumSize, CompletionHandler<void(Vector<Vector<Ref<API::TargetedElementInfo>>>&&)>&& completion)
```